elyra/pipeline/resources/properties.json

```diff
@@ -67,7 +67,7 @@
           "default": "Filename"
         },
         "description": {
-          "default": "The path to the notebook file.",
+          "default": "The path to the file.",
```

Comment:
I think this is dynamically handled in the server code

Reply:
I saw that, so I figured this a default/fallback value, so it should be generic. I changed this before finding that dynamic handling and just didn't change it back for that reason.